client/Components/GetInTouchForm.js

```diff
@@ -51,8 +57,8 @@ const GetInTouchForm = ( { handleFormChange }) => {
               type="text"
               required
               onFocus={(e) => {handleFormFieldFocus(e)}}
-              onBlur={(e) => {handleFieldBlur(e)}}/>
-              <div className="error-message" hidden={isHidden}>First and last, please.</div>
+              onBlur={(e) => {handleFieldBlur(e, e.target.name)}}/>
```

Comment:
In-line functions are memory expensive. A new function will be created every time the component renders. Giving synthetic events only the handler, i.e. `onFocus={handleFormFieldFocus}`, is equivalent to `onFocus={(e) => handleFormFieldFocus(e)}` except it refers to a single function with every render.